package.json

```diff
@@ -83,6 +86,7 @@
     "rollup-plugin-terser": "^5.3.0",
     "semver": "^7.3.2",
     "standard-version": "^8.0.0",
+    "tailwindcss": "^1.7.6",
```

Comment:
Do you want to pin to Tailwind v1.6 or are you ok with v1.7 here?

Reply:
this package is compatible with tailwind v1.7, so no need to pin it at 1.6.

Reply:
Sounds great!